web-rdfs/source/web_request_translator.cc

```diff
@@ -39,13 +39,24 @@ namespace webRequestTranslator {
 
     res += location;
     res += "\n";
-    res += "Content-Length: ";
-    res += std::to_string(contentOfFile.length());
+    res += "Content-Length: 0";
     res += "\n\n";
 
     return "";
   }
 
+  /**
```

Comment:
can get rid of this based on what I said about response->write in the create_file_handler